src/js/disability-benefits/components/AppealLayout.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Breadcrumbs from './Breadcrumbs';
+
+export default function AppealLayout({ children }) {
+  return (
+    <div>
+      <div>
+        <div className="row">
```

Comment:
Why is this nested in an otherwise empty `div`?

Reply:
Should the classes of the nested `div`s be consistent with other usages of Breadcrumbs?
```
<div>
  <div className="row">
    <div className="medium-12 columns">
      <Breadcrumbs/>
    </div>
  </div>
  <div className="row">
```

Reply:
I am not sure why I added that extra div. Probably can remove it.

@cehsu I was matching the way the Appeals app was using it. The layout is a little finicky, so I think I'd like to leave it the way it is since it looks right.

Reply:
I tried add the columns div, but it adds more padding and affects the alignment, so I'd have to correct for it again here, which doesn't seem like an improvement.